test/jasmine/tests/snapshot_test.js

```diff
@@ -245,5 +245,53 @@ describe('Plotly.Snapshot', function() {
                 done();
             });
         });
+
+        it('should handle quoted style properties', function(done) {
+            Plotly.plot(gd, [{
+                y: [1, 2, 1],
+                marker: {
+                    gradient: {
+                        type: 'radial',
+                        color: '#fff'
+                    },
+                    color: ['red', 'blue', 'green']
+                }
+            }], {
+                font: { family: 'Times New Roman' }
+            })
+            .then(function() {
+                d3.selectAll('text').each(function() {
+                    var tx = d3.select(this);
+                    expect(tx.style('font-family')).toEqual('\"Times New Roman\"');
+                });
+
+                d3.selectAll('.point').each(function() {
+                    var pt = d3.select(this);
+                    expect(pt.style('fill').substr(0, 6)).toEqual('url(\"#');
+                });
+
+                return Plotly.Snapshot.toSVG(gd);
+            })
+            .then(function(svg) {
+                var svgDOM = parser.parseFromString(svg, 'image/svg+xml');
+                var i;
+
+                var textElements = svgDOM.getElementsByTagName('text');
+                expect(textElements.length).toEqual(11);
+
+                for(i = 0; i < textElements.length; i++) {
+                    expect(textElements[i].style['font-family']).toEqual('\"Times New Roman\"');
+                }
+
+                var pointElements = svgDOM.getElementsByClassName('point');
+                expect(pointElements.length).toEqual(3);
```

Comment:
Before this patch, `pointElement.length` was 0 i.e. `Snapshot.toSVG` failed to process them.